packages/tools/test/getVolumeId_test.js

```diff
@@ -0,0 +1,13 @@
+import { getVolumeId } from '../../tools/src/utilities/getVolumeId';
+
+describe('getVolumeId', () => {
+  it('should extract volumeId from target', () => {
```

Comment:
Can we add this test with arguments, eg volumeId:aaaa?sliceIndex=4&focalPoint=3,2,5.5
It should still extract aaaa

Reply:
Done